src/plugins/dashboard/public/application/utils/use/use_dashboard_app_state.tsx

```diff
@@ -193,6 +193,17 @@ export const useDashboardAppAndGlobalState = ({
 
         subscriptions.add(stopSyncingFromTimeFilters);
 
+        const stopSyncingFromGlobalFilters = filterManager.getUpdates$().subscribe(() => {
```

Comment:
Shouldnt we unsubscribe from this when exiting the hook?

Reply:
i did unsubscribe by `subscriptions.add(stopSyncingFromGlobalFilters);` and `subscriptions.unsubscribe();`